src/util/system.h

```diff
@@ -150,9 +157,9 @@ class ArgsManager
     std::string m_network GUARDED_BY(cs_args);
     std::set<std::string> m_network_only_args GUARDED_BY(cs_args);
     std::map<OptionsCategory, std::map<std::string, Arg>> m_available_args GUARDED_BY(cs_args);
-    std::set<std::string> m_config_sections GUARDED_BY(cs_args);
+    std::list<SectionInfo> m_config_sections GUARDED_BY(cs_args);
 
-    NODISCARD bool ReadConfigStream(std::istream& stream, std::string& error, bool ignore_invalid_keys = false);
+    NODISCARD bool ReadConfigStream(std::istream& stream, const std::string& label, std::string& error, bool ignore_invalid_keys = false);
```

Comment:
I thought it might not only be a file stream. (unit test uses ```std::istringstream```).
But now I agree that ```filepath``` is better than ```label```.  Fixed.